client/my-sites/comments/comment-list/index.jsx

```diff
@@ -17,6 +17,7 @@ import {
 	uniq
 } from 'lodash';
 import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+import page from 'page';
```

Comment:
We can choose the defaultExport name here:
`import pageRouter from 'page'`;

For other cases we can use an alias:
```
import { foo as bar } from 'baz';
```

Reply:
For anyone picking this up, please alias the page call. I think the diff  will end up being much smaller then.

Reply:
Completely removed `page` from the components to keep them side-effect free on feb6c04c7fcf2559add206fc94f96634823f6d26.